src/components/vitals/AnalyticsMetricsCard.vue

```diff
@@ -0,0 +1,66 @@
+<template>
+  <KCard
+    class="mb-3 analytics-overview"
+    data-testid="analytics-metrics-overview"
+  >
+    <template #body>
+      <MetricsProvider
+        :additional-filter="additionalFilter"
```

Comment:
This TypeError is saying that you're passing a raw value when the prop expects a ref.  `MetricsProviderInternal` no longer needs this to be passed as a ref; perhaps fixing that type in `MetricsProvider` will fix a few errors.